bands/site_sections/band_admin.py

```diff
@@ -12,10 +12,11 @@ def _required_message(self, params, fields):
                 'is' if len(missing) == 1 else 'are')
         return ''
 
-    def index(self, session, message=''):
+    def index(self, session, message='', filter='show-all'):
```

Comment:
The new `filter` property (and the associated menu items) allows different departments to have their own workflows, so our Guest relationships head isn't constantly working around bands in the band admin list.